packages/daimo-common/src/daimoPay.ts

```diff
@@ -13,18 +13,22 @@ import z from "zod";
 
 import { BigIntStr, zAddress } from "./model";
 
-// lifecycle: waiting payment -> processed.
+// lifecycle: waiting payment -> pending processing -> start submitted -> processed (onchain tx was successful)
+// strings are not consistent for legacy reasons
 export enum DaimoPayOrderStatusSource {
   WAITING_PAYMENT = "waiting_payment",
   PENDING_PROCESSING = "pending_processing",
-  PROCESSED = "processed",
+  START_SUBMITTED = "start_submitted",
+  START_SUCCESSFUL = "processed",
```

Comment:
recommend keeping same names, confusing if different in codebase vs db

however, we can add docstring, eg

```ts
  /** Start transaction successful. Bridge initiated, if applicable. */
  PROCESSED = "processed",
```